src/BloomBrowserUI/pageChooser/js/page-chooser.ts

```diff
@@ -111,8 +135,8 @@ class PageChooser {
             var dataBookArray = $( "div[data-book='bookTitle']", pageData );
             var collectionTitle = $( dataBookArray.first() ).text();
             // Add title and container to dialog
-            var collectionToAdd = $( collectionHTML).clone();
-            $( collectionToAdd ).find( ".collectionCaption" ).text(collectionTitle);
+            var collectionToAdd = $(collectionHTML).clone();
+            this.setLocalizedText($(collectionToAdd).find(".collectionCaption"), 'Template.', collectionTitle);
```

Comment:
Well, you know how I feel about, essentially, perpetuating mistakes by saying well it was already there. We just go downhill. Each thing we do should see to improve the situation.